scripts/gui/microbe_hud.mjs

```diff
@@ -14,13 +14,19 @@ let wonOnce = false;
 // Variable to show data useful during develop
 const showMouseCoordinates = false;
 
+// For toggling paused with the pause button
+let paused = false;
```

Comment:
This doesn't get reset when starting a new game. So if you open the menu in a new game, then go back to the main menu and start a new game, the game starts paused (and the menu button looks like it is active). So exiting to menu should clear the game being paused.